src/components/Controls/index.tsx

```diff
@@ -204,10 +191,27 @@ const Controls = ({
         showControlsFor(2);
         setCurrVideoTime(video.currentTime);
       };
-      document.addEventListener("keypress", listener);
+      document.addEventListener("keypress", keypressListener);
+
+      // The keypress listeners are different from the keydown listeners.
+      const keydownListener: (e: KeyboardEvent) => void = (e: KeyboardEvent) => {
+        if (e.key === "ArrowUp") {
+          if (volume < 95) updateVolume(volume + 5);
+          else updateVolume(100);
+        } else if (e.key === "ArrowDown") {
+          if (volume > 5) updateVolume(volume - 5);
+          else updateVolume(0);
+        } else if (e.key === "ArrowRight") {
+          video.currentTime += 5;
+        } else if (e.key === "ArrowLeft") {
+          video.currentTime -= 5;
+        }
+      }
+      document.addEventListener("keydown", keydownListener);
```

Comment:
Why new listeners is added?

What is the purpose of the new listener, which is extracted from the `keypress` listener?
Why do we need the `keydown` listener?

Reply:
- This is because, the `keypress` listener is only correctly handling the printable keys from the keyboard, but sometimes neglecting the arrowKeys and other keys.
- But they work very fine with the `keydown` or the `keyup` listener. So extracted this as a new listener and added it.